controllers/shipwrightbuild_controller_test.go

```diff
@@ -0,0 +1,173 @@
+package controllers
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	o "github.com/onsi/gomega"
+	"github.com/shipwright-io/operator/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
```

Comment:
Should be sorted into blocks

Reply:
Okay, I'm adding a extra plugin to my editor, so that should happen automatically from now on 😅. Please consider latest changes.